mesonbuild/modules/cmake.py

```diff
@@ -290,7 +298,8 @@ def write_basic_package_version_file(self, state, _args, kwargs):
 
         conf = {
             'CVF_VERSION': (version, ''),
-            'CMAKE_SIZEOF_VOID_P': (str(self.detect_voidp_size(state.environment)), '')
+            'CMAKE_SIZEOF_VOID_P': (str(self.detect_voidp_size(state.environment)), ''),
+            'CVF_ARCH_INDEPENDENT': (arch_independent, '')
```

Comment:
For future proofing in case we ever need to add another line here, we can leave a trailing comma on this line too.

This is not some terrible language like json, where trailing commas are a syntax error. :D